README.md

```diff
@@ -6,13 +6,11 @@ This library provides a basic interface for (splittable) random number generator
 
 The API documentation can be found here:
 
-   http://hackage.haskell.org/package/random/docs/System-Random.html
+> http://hackage.haskell.org/package/random/docs/System-Random.html
 
 A module supplying this interface is required for Haskell 98 (but not Haskell
-2010). An older [version]
-(http://www.haskell.org/ghc/docs/latest/html/libraries/haskell98/Random.html)
+2010). An older [version](http://www.haskell.org/ghc/docs/latest/html/libraries/haskell98/Random.html)
```

Comment:
Error 404. 
Should be https://downloads.haskell.org/~ghc/7.6.3/docs/html/libraries/haskell98/Random.html

Reply:
Good catch! Added to the PR.